slang_flutter/lib/slang_flutter.dart

```diff
@@ -158,8 +157,7 @@ class _TranslationProviderState<E extends BaseAppLocale<E, T>,
   }
 }
 
-class InheritedLocaleData<E extends BaseAppLocale<E, T>,
-    T extends BaseTranslations<E, T>> extends InheritedWidget {
+class InheritedLocaleData<E extends BaseAppLocale<E, T>, T extends BaseTranslations<E, T>> extends InheritedWidget {
```

Comment:
While, I also like this more, `dart format` gives a different result

Reply:
Seems my IDE uses `dart format --line-length 120`. Shall I `dart format` to modify it back?

Reply:
Yes, you can do that :)

Reply:
done